setup.py

```diff
@@ -1,8 +1,8 @@
 from setuptools import setup
 
 install_requires = [
-    "huggingface_hub==0.8.1",
-    "pyyaml==6.0",
+    "huggingface_hub~=0.8.1",
```

Comment:
For huggingface_hub I'm not sure the tilde is useful since there is no version 0.8 above 0.8.1

Reply:
oh, I meant `~=0.8` (if hf follows semantic versioning)

Reply:
0.8.0 was yanked as there was a backwards compatible issue so it's not installable. 0.8.1 is the only version in the 0.8 range, so adding this change would not contribute here

Reply:
I meant `~=0.8` to allow `0.9`, `0.x.x`

Reply:
Ah I see! Could you modify that in the PR please? :pray: